lib/portage/repository/config.py

```diff
@@ -177,6 +179,13 @@ def __init__(self, name, repo_opts, local_config=True):
 		else:
 			masters = None
 
+		self.maintainer_nag = repo_opts.get('maintainer-nag', 'yes').lower() == 'yes'
```

Comment:
We have to disable it by default here, since we can't assume that third-party repos will have maintainers listed in metadata.xml.

Reply:
Meanwhile, we need to make sure that we respect a default from layout.conf if it's set there.

Reply:
Fixed the default locally, need to figure out why I can't get it to respect layout.conf defaults at the moment.